src/main/java/com/scalesec/vulnado/LoginController.java

```diff
@@ -0,0 +1,44 @@
+package com.scalesec.vulnado;
+
+import org.springframework.boot.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.boot.autoconfigure.*;
+import org.springframework.stereotype.*;
+import org.springframework.beans.factory.annotation.*;
+import java.io.Serializable;
+
+@RestController
+@EnableAutoConfiguration
+public class LoginController {
+  @Value("${app.secret}")
+  private String secret;
+
+  @CrossOrigin(origins = "*")
+  @RequestMapping(value = "/login", method = RequestMethod.POST, produces = "application/json", consumes = "application/json")
+  LoginResponse login(@RequestBody LoginRequest input) {
```

Comment:
Detected a method annotated with 'RequestMapping' that does not specify the HTTP method. CSRF protections are not enabled for GET, HEAD, TRACE, or OPTIONS, and by default all HTTP methods are allowed when the HTTP method is not explicitly specified. This means that a method that performs state changes could be vulnerable to CSRF attacks. To mitigate, add the 'method' field and specify the HTTP method (such as 'RequestMethod.POST').